UnityProjects/MRTKDevTemplate/Assets/XR/Settings/OpenXR Package Settings.asset

```diff
@@ -410,6 +411,36 @@ MonoBehaviour:
   company: Unity
   priority: 0
   required: 0
+--- !u!114 &-243070923169931762
```

Comment:
Based on our discussion today, should we hold off on applying this change, as the MR OpenXR Plugin does not work with the new Meta Quest feature name (broken manifest, etc)? @keveleigh

Reply:
Works for me! That update is available in #11303, so I'll mark this as DO NOT MERGE until that PR goes in.

Reply:
@Zee2 This one should be ready now!